app/views/tagging_spreadsheets/index.html.erb

```diff
@@ -12,6 +12,7 @@
       <th></th>
       <th></th>
     </tr>
+    <%= render partial: 'shared/table_filter' %>
```

Comment:
This is the correct way to call partials: http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials